server/server.go

```diff
@@ -129,7 +129,7 @@ func (s *Server) newConn(conn net.Conn) *clientConn {
 	return cc
 }
 
-func (s *Server) skipAuth() bool {
+func (s *Server) isLocal() bool {
```

Comment:
The function name is `isLocal` and the function implement is `c.cfg.Socket` ? That seems weird 
@jackysp

Reply:
Should it be `isLocalSocket` or `isUnixSocket`?  There could be a local `tcp` connection.  We will not support them, but MySQL also supports local named pipe and shm on windows.

Reply:
`isUnixSocket` sounds good.